python/orca/test/bigdl/orca/learn/ray/ctx/conftest.py

```diff
@@ -0,0 +1,41 @@
+#
```

Comment:
I don't think we need this file. There are already setup methods in each test file.

Reply:
Still need `conf = {"spark.python.worker.reuse": "false"}`.

Reply:
Removed, worth trying 😃